### PR TITLE
CR-1120378 : RPM packager using wrong python version (was: xbtop doesn't run on CentOS 7)

### PR DESCRIPTION
**Pull request by rozumx**
Debugging issue with regards to with enabling python3 scripts to be packaged with CMake.  In this case, it is using the python2 to validate that the sources are valid (instead of python3).

#### Problem solved by the commit
When packaging a RPM package on CentOS, the wrong version of python (e.g., python2) was being used to compile (e.g., validate) the python scripts.   For this reason, CentOS packaging of xbtop was previously disabled.

This PR both addresses this issue (e.g., python3 is now used) and re-enables packaging the xbtop for CentOS packages.

#### Bug / issue (if any) fixed, which PR introduced the bug, how it was discovered
CR-1120378

#### How problem was solved, alternative solutions (if any) and why they were rejected
Updated CPack to instruct the rpm packager to use python3.

#### Risks (if any) associated the changes in the commit
None

#### What has been tested and how, request additional testing if necessary
Manual testing the the XRT-Pipeline

#### Documentation impact (if any)
n/a

### CODE REVIEW

File: src/CMake/cpackLin.cmake [hunk 1]
```diff
@@ -19,6 +19,9 @@ SET(CPACK_ARCHIVE_COMPONENT_INSTALL ON)
 SET(CPACK_DEB_COMPONENT_INSTALL ON)
 SET(CPACK_RPM_COMPONENT_INSTALL ON)
 
+# Force python byte compilation to use python3
```
Comment (stsoe) [inline src/CMake/cpackLin.cmake:22]: I think your commit description is better than this comment line.
I don't really know what this line does by looking at it, but you explained it in the description.  Could you add more details here?

**Comment by rozumx** [comment]
retest this please

**Review by AShivangi — APPROVED** [review]
@uday610, we are reenabling xbtop on CenOS. Please keep an eye on this PR and update the documentation accordingly.

**Review by stsoe — APPROVED** [review]

**Comment by uday610** [comment]
@AShivangi , so we dont need to mention any limitation (that needed to copy python file etc) for Centos right?

**Comment by dayeh-xilinx** [comment]
retest this please.

**Pull request merged**